src/flows/zome-call.ts

```diff
@@ -1,40 +1,54 @@
 
+import * as Config from '../config'
 import {HappID} from '../types'
 import {
-  errorResponse,
-  fail,
+  lookupHoloInstance,
+  instanceIdFromAgentAndDna,
   serviceLoggerInstanceIdFromHappId,
-  zomeCallByDna,
   zomeCallByInstance,
   zomeCallSpec,
 } from '../common'
 
+import {lookupDnaByHandle} from './install-happ'
```

Comment:
BTW - shouldn't this be somewhere in the utils/helpers module? Lookup call within zomeCall doesn't seem like install-specific func ;)

Reply:
It's true - I think there may be some circular dependency here, or maybe I just got lazy?

Reply:
Well, such things happen all the time, since software is a living thing and it's hard to think about the architecture with each small change. 😄

Reply:
`lookupDnaByHandle` is only needed for the ZomeCall flow.  What do you think about relocating it into ZomeCall flow file instead of a general utils module, to keep the calls specific to their context?